corehq/apps/export/views/download.py

```diff
@@ -551,8 +552,12 @@ def get(self, request, domain, export_instance_id):
         export_instance = get_properly_wrapped_export_instance(export_instance_id)
         assert domain == export_instance.domain
         output_file = BytesIO()
-        generate_from_export_instance(export_instance, output_file)
-        output_file.seek(0)
-        response = HttpResponse(output_file, content_type='application/vnd.ms-excel')
-        response['Content-Disposition'] = f'attachment; filename="{export_instance.name}.xlsx"'
-        return response
+        try:
+            generate_from_export_instance(export_instance, output_file)
+        except DETConfigError as e:
+            return HttpResponse(_(f'Sorry, something went wrong creating that file: {e}'))
+        else:
```

Comment:
nit: else not necessary since the except clause returns.

Reply:
https://github.com/dimagi/commcare-hq/pull/27040/commits/d37bf38b1f13bf97f72026ad82885ea13aeffc37